### DISCUSSION TITLE
Improvement: user配下のページのデフォルト公開設定を自分のみにする機能がほしい

### DISCUSSION THREAD

**Discussion by you-at4** [growilabs/growi#7747] [Ideas / Requests]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        |Ubuntu 18.04|
|GROWI     |3.4.5|
|node.js   |10.15.3|
|npm       |6.9.0|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

### Client

| item     | version |
| ---      | --- |
|OS        |windows10|
|browser   |Chrome 73|

How to reproduce? (再現手順)
---------------------------
1. ユーザー(`user1`)が新規ページ `/user/user1/メモ/2019/01/01/NewMemo` を作成

What is the expected result? (期待される動作)
-------------------------------------------
`/user/user1/メモ/2019/01/01/NewMemo` の公開設定が自分のみになっている

Note
----
最初の操作として`/user/該当ユーザー/メモ`を作成して公開設定を自分のみにするように案内していますが、デフォルトで設定できるほうがありがたいです。
ユーザー追加時に`/user/(user名)/メモ`を自分のみの公開設定で自動生成するようになればいいかと思います。
(Anyone with the linkでもいいかも)

**Comment by yuki-takei** [comment]
@you-at4 ご要望ありがとうございます。

現在いただいている

> ユーザー(user1)が新規ページ /user/user1/メモ/2019/01/01/NewMemo を作成

を行った時に

> /user/user1/メモ/2019/01/01/NewMemo の公開設定が自分のみになっている

という挙動については、必ずしも全ての GROWI サイトにとってベターとは限らないと思っています。
なので、やるとしたら管理画面で選択できるように、という感じになるとは思います。

## 案1

1. 管理画面で、「デフォルト公開設定を変更するパスを登録」という機能を設ける
1. 上記機能で「/user」を登録し、Just Me にセットする

このような挙動だとどうでしょうか？
ご意見お願いいたします。

**Comment by you-at4** [comment]
そのような機能でいいと思います。
最初はユーザー毎に/user配下のページの公開設定を変更できればいいのではないかと思ったのですが
管理画面で全体を一括で設定できたほうが運用的にも楽だと思うので。
